input/resources/au-core-responder.xml

```diff
@@ -866,6 +866,7 @@
 <blockquote>
 <div>
 <p>If the responder supports the Encounter resource, the responder <strong>SHALL</strong> support the AU Core profile and the conformance expectations for the Encounter resource.</p>
+<p>The Encounter resource can communicate the clinical indication using either a code in Encounter.reasonCode or a reference using Encounter.reasonReference. Although both <code>Encounter.reasonCode</code> and <code>Encounter.reasonReference</code> are marked as Must Support, a responder is not required to support both but <strong>SHALL</strong> support at least one of these elements. The requester <strong>SHALL</strong> support both elements.</p>
```

Comment:
Apply formatting to all instances of Encounter.reasonCode and Encounter.reasonReference.

Reply:
lol, don't need to repeat a statement on 'all instances of'

Reply:
Fix not included in this PR.

Reply:
I can see it in the PR.

Reply:
fixed.